source/AsyncRgbLedSimulationDataGenerator.h

```diff
@@ -19,11 +22,20 @@ class AsyncRgbLedSimulationDataGenerator
 	U32 mSimulationSampleRateHz;
 
 protected:
-	void CreateSerialByte();
-	std::string mSerialText;
-	U32 mStringIndex;
+	void CreateRGBWord();
+	RGBValue RandomRGBValue() const;
+
+    void WriteRGBTriple( const RGBValue& rgb );
+	void WriteUIntData( U32 data, U8 bit_count );
+	void WriteBit(bool b);
```

Comment:
let's change the signature from bool to BitState

Reply:
Will do.